### PR TITLE
Changes to dynamic block creation process

### PR DESCRIPTION
**Pull request by SmVladyslav**
## Purpose of this pull request
This pull request (PR) adds clarifications for creating a dynamic block

## Affected DevDocs pages
https://docs.magento.com/user-guide/cms/dynamic-blocks.html

### CODE REVIEW

File: src/cms/dynamic-blocks.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ _Page Builder - dynamic block with price rule and customer segment_
    ![]({% link images/images-ee/page-builder-dynamic-block-customer-segment.png %}){: .zoom}
    _Choosing a customer segment_
 
+   {:.bs-callout-info}
+   - If no **Segment** is created then **Dynamic Block** will be visible to everyone.
```
Comment (hickskenh) [inline src/cms/dynamic-blocks.md:48]:
```suggestion
   - If no **Segment** is created, the **Dynamic Block** will be visible to everyone.
```

File: src/cms/dynamic-blocks.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ _Page Builder - dynamic block with price rule and customer segment_
...
+   - If **Customer** does not belong to any of **Segments** and **Dynamic Block** is created for All Segments then contents of **Dynamic Block** will still be displayed.
+   - If all **Customer Segments** that were assigned to **Dynamic Block** are deleted then its contents will be visible to everyone.
```
Comment (hickskenh) [inline src/cms/dynamic-blocks.md:50]:
```suggestion
   - If all **Customer Segments** that were assigned to **Dynamic Block** are deleted, its contents will be visible to everyone.
```

File: src/cms/dynamic-blocks.md [hunk 3]
```diff
@@ -44,6 +44,11 @@ _Page Builder - dynamic block with price rule and customer segment_
    ![]({% link images/images-ee/page-builder-dynamic-block-customer-segment.png %}){: .zoom}
    _Choosing a customer segment_
 
+   {:.bs-callout-info}
+   - If no **Segment** is created then **Dynamic Block** will be visible to everyone.
+   - If **Customer** does not belong to any of **Segments** and **Dynamic Block** is created for All Segments then contents of **Dynamic Block** will still be displayed.
```
Comment (hickskenh) [inline src/cms/dynamic-blocks.md:49]:
```suggestion
   - If **Customer** does not belong to any **Segments** and the **Dynamic Block** is created for All Segments, the contents of **Dynamic Block** will still be displayed.
```

**Review by hickskenh — CHANGES_REQUESTED** [review]
requesting grammatical and punctuation corrections and resubmit for review

**Comment by SmVladyslav** [comment]
Corrected

**Review by hickskenh — APPROVED** [review]
approved

**Comment by hickskenh** [comment]
running tests

**Comment by leslietilling** [comment]
Hello @SmVladyslav  -- we are not allowed to merge unless you sign the Adobe CLA: http://opensource.adobe.com/cla.html

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**